### ISSUE TITLE
MD029 (ol-prefix) and nested unordered lists in ordered lists

### ISSUE THREAD

**Issue by EvgenyOrekhov** [DavidAnson/markdownlint#38]
If I use the following rule:

```json
{
    "ol-prefix": {
        "style": "ordered"
    }
}
```

then markdownlint says "Ordered list item prefix (Expected: 1; Actual: 2)" to the following code:

```md
1. a
- b
2. c
- d
```

It looks like markdownlint thinks that `2. c` is the beginning of a new list, but it's the second item of the single ordered list.

If I indent the unordered lists, markdownlint says "Consider starting bulleted lists at the beginning of the line".

What is the correct way of using nested unordered lists in ordered lists?

**Comment by DavidAnson** [comment]
When indenting, did you use three spaces? The following should do what you want:

```
1. a
   - b
2. c
   - d
```

Markers for nested lists should begin under the text of the previous item - for numbered lists that's typically 3 vs. 2 spaces.

You can play around interactively here to see that `markdown-it` doesn't recognize the nested lists with just 2 spaces: https://dlaa.me/markdownlint/

**Comment by EvgenyOrekhov** [comment]
@DavidAnson thanks!
